client/actions/planning/api.js

```diff
@@ -868,6 +888,29 @@ const lockFeaturedPlanning = () => (
     )
 );
 
+
+const fetchFeaturedPlanningItemById = (id) => (
+    (dispatch, getState, {api}) => api.find('planning_featured', id).then((item) => item)
+);
+
+/**
+ * Action dispatcher that attempts to lock a Planning item through the API
+ * @param {object} item - The Planning item to lock
+ * @return Promise
+ */
+const saveFeaturedPlanning = (updates) => (
+    (dispatch, getState, {api}) => {
+        const item = selectors.featuredPlanning.featuredPlanningItem(getState()) || {};
+
+        return api('planning_featured').save(cloneDeep(item), {
+            ...(pickBy(cloneDeep(item), (v, k) => (!k.startsWith('_')))),
+            ...updates,
+            tz: moment.tz.guess(),
```

Comment:
I believe this should use the configured default timezone from the config and not the client browser's timezone.
i.e. `config.defaultTimezone`